src/main/cliapp/src/components/datatables/DiseaseAnnotationsComponent.js

```diff
@@ -76,11 +76,10 @@ export const DiseaseAnnotationsComponent = () => {
     );
 
     useQuery(['diseaseRelationTerms'],
-        () => controlledVocabularyService.getTerms('disease_relation_terms'), {
-            onSuccess: (data) => {
-                setDiseaseRelationTerms(data)
-            }
-        }
+        () => searchService.search("vocabularyterm", 15, 0, null, {"vocabFilter": {"vocabulary.name": "Disease Relation Vocabulary"}})
+        .then((data) => {
+            setDiseaseRelationTerms(data.results);
+        })
```

Comment:
@abecerra this works. However some vocab's can't be selected this way. So it would be better to use the controlledVocabularyService and pass in "Disease Relation Vocabulary" to the `getTerms` method.

Reply:
Ok!

Reply:
@oblodgett I helped andreas with this, I thought we talked about switching out the controlledVocabService for the searchService? Or do you mean put a call to the search service within the ConrolledVocabService?

Reply:
Ya... you are right.. I think the issue with promises etc is going to be a problem.